archey/logos/darwin.py

```diff
@@ -38,21 +38,17 @@
 ]
 
 LOGO_RETRO = [
-    """{c[0]}                     {c[1]}'c.{c[0]}          """,
-    """{c[0]}                  {c[1]},xNMM.{c[0]}          """,
-    """{c[0]}                {c[1]}.OMMMMo{c[0]}           """,
-    """{c[0]}                {c[1]}OMMM0,{c[0]}            """,
-    """{c[0]}      {c[1]}.;loddo:'{c[0]} {c[1]}loolloddol;.{c[0]}      """,
-    """{c[0]}    {c[1]}cKMMMMMMMMMMNWMMMMMMMMMM0:{c[0]}    """,
-    """{c[0]}  {c[2]}.KMMMMMMMMMMMMMMMMMMMMMMMWd.{c[0]}    """,
-    """{c[0]}  {c[2]}XMMMMMMMMMMMMMMMMMMMMMMMX.{c[0]}      """,
-    """{c[0]}  {c[3]}MMMMMMMMMMMMMMMMMMMMMMMM:{c[0]}       """,
-    """{c[0]}  {c[3]}MMMMMMMMMMMMMMMMMMMMMMMM:{c[0]}       """,
-    """{c[0]}  {c[4]}MMMMMMMMMMMMMMMMMMMMMMMMX.{c[0]}      """,
-    """{c[0]}  {c[4]}kMMMMMMMMMMMMMMMMMMMMMMMMWd.{c[0]}    """,
-    """{c[0]}  {c[5]}.XMMMMMMMMMMMMMMMMMMMMMMMMMMk{c[0]}   """,
-    """{c[0]}   {c[5]}.XMMMMMMMMMMMMMMMMMMMMMMMMK.{c[0]}   """,
-    """{c[0]}     {c[6]}kMMMMMMMMMMMMMMMMMMMMMMd{c[0]}     """,
-    """{c[0]}      {c[6]};KMMMMMMMWXXWMMMMMMMk.{c[0]}      """,
-    """{c[0]}        {c[6]}.cooc,.{c[0]}    {c[6]}.,coo:.{c[0]}        """
+    """{c[0]}                {c[1]}###{c[0]}               """,
+    """{c[0]}              {c[1]}####{c[0]}                """,
+    """{c[0]}              {c[1]}###{c[0]}                 """,
+    """{c[0]}      {c[1]}#######    #######{c[0]}          """,
```

Comment:
For "inverted-selection" consistency :

```suggestion
    """{c[0]}      {c[1]}#######{c[0]}    {c[1]}#######{c[0]}          """,
```